### ISSUE TITLE
[test-triage] chip_sw_otbn_ecdsa_op_irq

### ISSUE THREAD

**Issue by engdoreis** [lowRISC/opentitan#14219]
### Hierarchy of regression failure

Chip Level

### Failure Description
```
UVM_INFO @ 2164.235924 us: (sw_test_status_if.sv:51) [tb.u_sim_sram.u_sim_sram_if.u_sw_test_status_if] SW test transitioned to SwTestStatusInBootRom.
UVM_INFO @ 2663.735924 us: (sw_logger_if.sv:365) [tb.u_sim_sram.u_sim_sram_if.u_sw_logger_if.construct_log_and_print.unnamed$$_0.unnamed$$_1.unnamed$$_2.isolation_thread.unnamed$$_0.unnamed$$_1] String arg at addr ff60: ff60
UVM_INFO @ 2663.735924 us: (sw_logger_if.sv:519) [test_rom_sim_dv(sw/device/lib/testing/test_rom/test_rom.c:88)] ff60
UVM_INFO @ 2668.205924 us: (sw_logger_if.sv:519) [test_rom_sim_dv(sw/device/lib/testing/test_rom/test_rom.c:137)] Test ROM complete, jumping to flash!
UVM_INFO @ 2679.365924 us: (sw_test_status_if.sv:51) [tb.u_sim_sram.u_sim_sram_if.u_sw_test_status_if] SW test transitioned to SwTestStatusInTest.
UVM_INFO @ 5492.455924 us: (sw_logger_if.sv:365) [tb.u_sim_sram.u_sim_sram_if.u_sw_logger_if.construct_log_and_print.unnamed$$_0.unnamed$$_1.unnamed$$_2.isolation_thread.unnamed$$_0.unnamed$$_1] String arg at addr 200031d9: Initialization
UVM_INFO @ 5492.715924 us: (sw_logger_if.sv:519) [otbn_ecdsa_op_irq_test_prog_sim_dv(sw/device/tests/otbn_ecdsa_op_irq_test.c:233)] Initialization took 280811 cycles or 2808 us @ 100 MHz.
UVM_INFO @ 5499.225924 us: (sw_logger_if.sv:519) [otbn_ecdsa_op_irq_test_prog_sim_dv(sw/device/tests/otbn_ecdsa_op_irq_test.c:393)] Signing
```
I'm assuming the error is a Timeout 

### Steps to Reproduce

- Commit hash where failure was observed
  - fc9ce22c1c836951396af5af0cb9306ca9ba09f8
- dvsim invocation command to reproduce the failure, inclusive of build and run seeds
  -   ./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i chip_sw_otbn_ecdsa_op_irq

### Tests with similar or related failures

- [ ] chip_sw_otbn_ecdsa_op_irq_jitter_en

**Comment by vogelpi** [comment]
I had a look at this. It's indeed a timeout. First of all, we define a simulation time timeout in `hw/top_earlgrey/dv/chip_sim_cfg.hjson` with `run_opts: ["+sw_test_timeout_ns=28_000_000"]`. After increasing that, the test still fails due to a wall clock timeout after 120 minutes on my machine. I need to rerun this without timeout and see how long it actually takes.

**Comment by sriyerg** [comment]
Hi @vogelpi, in addition to the simulation timeout for the SW, we recently also added a wall clock timeout (i.e. kill the simulation externally if it exceeds N minutes). The default timeout value is 60 minutes. 

There were some other tests that also failed due to the wall clock timeout. I ended up finding the worst case wall clock time for ALL tests that failed due to timeout (including these ones) - I will have a PR out soon.
